source/core/index.ts

```diff
@@ -150,6 +149,15 @@ export interface Options extends URLOptions, SecureContextOptions {
 	socketPath?: string;
 	method?: Method;
 	createConnection?: (options: http.RequestOptions, oncreate: (error: Error, socket: Socket) => void) => Socket;
+
+	// From tls.ConnectionOptions
```

Comment:
```suggestion
	// From `tls.ConnectionOptions`
```

Reply:
Some line before there's `// From http.RequestOptions` which was already there, shoud I also change it?

Reply:
Yup